### ISSUE TITLE
[Quest] 26587 | Breaking Out is Hard to Do - Redridge Mountains

### ISSUE THREAD

**Issue by iTz-AddriX** [WoWManiaUK/Blackwing-Lair#4333]
**Links:** 
1. Quest - [Breaking Out is Hard to Do](https://www.wowhead.com/quest=26587/breaking-out-is-hard-to-do)
2. NPC - [Blackrock Worg Captain](https://www.wowhead.com/npc=43341/blackrock-worg-captain)
3. NPC - [Blackrock Battle Worg](https://www.wowhead.com/npc=43341/blackrock-worg-captain)
4. Item - [Messner's Cage Key](https://www.wowhead.com/item=58950/messners-cage-key)

**What is happening:** There are too many [Dogs](https://www.wowhead.com/npc=43341/blackrock-worg-captain) surrounding [Messner's Cage Key](https://www.wowhead.com/item=58950/messners-cage-key) its way to hard.

**What should happen:** The [Dogs](https://www.wowhead.com/npc=43341/blackrock-worg-captain) should stay asleep unless alerted by [Captain](https://www.wowhead.com/npc=43341/blackrock-worg-captain)

**Proof:**
![WoWScrnShot_040220_173515](https://user-images.githubusercontent.com/62844627/78302242-656c3580-7508-11ea-8714-3b533d4322ed.jpg)

**Comment by Echinaceax** [comment]
I dont think its the fact there is too many dogs, same on video 
https://www.wowhead.com/quest=26587/breaking-out-is-hard-to-do#videos:id=84604

Its it that the dogs are waking up @iTz-AddriX ?

**Comment by iTz-AddriX** [comment]
@Echinaceax Yes, the dogs wake if you get so close to them they become aggro. There is just many to take down with out be able to sneak past them all.

**Comment by Echinaceax** [comment]
Confirmed
Dogs are not working as they should be, they awaken even when **Blackrock Worg Captain** hasnt see you to alert the dogs
